atom/index.d.ts

```diff
@@ -81,9 +81,9 @@ export interface ReadableAtom<Value = any> {
    * Low-level method to read store’s value without calling `onStart`.
    *
    * Try to use only {@link ReadableAtom#get}.
-   * Without subscribers, value can de undefined.
+   * Without subscribers, value can be undefined.
    */
-  readonly value: undefined | Value
+  readonly value: undefined extends Value ? undefined | Value : Value
```

Comment:
Will it work with `computed()`? I think `computed` will have `store.value === undefined` before the subscriber even if the origin type has no `undefined`

Reply:
Yep, it won't because [`undefined` is explicitly set as initial value for `atom` in `computed`](https://github.com/nanostores/nanostores/blob/main/computed/index.js#L30). Looks like we can set `stores.map(store => store.value)` instead, but it will work only for sync callbacks.

Reply:
We can add some extra type like `PreinitializedAtom`

Reply:
BTW, you can use `$store.get()` to avoid `?`

Reply:
Added `PreinitializedAtom` util as you suggested.

> BTW, you can use $store.get() to avoid ?

But it will cause subscription, isn't it?

Reply:
> But it will cause subscription, isn't it?

Only if there is no other listeners.

> Added PreinitializedAtom util as you suggested.

Did you push? Don't see in PR

Reply:
Sorry :) Now it is pushed